### DISCUSSION TITLE
Sanırım Biraz uzun sürecek.

### DISCUSSION THREAD

**Discussion by ghost** [IBAX-io/go-ibax#1331] [Soru Cevap - Türkçe]
1 Şubatta airdrop dağatıcaklardır ama Proje için hiç bir gelişme olamyacaktır yani airdoplarını satmak isteyen kitleye satmak mantıklı değil demek istiyorum fakat isteselerde satamıcaklarını bilmelerini sitiyorum.  ( benim düşüncem 👍 )

**Comment by Skips1** [comment]
Zaten herkes bunu düşünüyodur kanka

**Comment by Omertastan2036** [comment]
Daha piyasaya çıkacak

**Comment by natolu2738** [comment]
Hodle o halde. Hedef 10$
